Ports/c-ares/package.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='c-ares'
+version='1.19.0'
+files="https://c-ares.org/download/c-ares-${version}.tar.gz c-ares-${version}.tar.gz bfceba37e23fd531293829002cac0401ef49a6dc55923f7f92236585b7ad1dd3"
+auth_type='sha256'
+useconfigure='true'
+configopts=("-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt")
+
+configure() {
+	mkdir -p c-ares-build
+	cmake -G Ninja \
+		"${configopts[@]}" \
+		-S "c-ares-${version}" \
```

Comment:
Tim can correct me if I'm misremembering, but this should be handled by `workdir`, yes?

`run cmake <blah>` chould `cd` to the workdir and handle the -S part at least.

Also there's tabs in this file, pls convert to 4 spaces.

Reply:
Sorry about the tabs! Fixed that in the latest push. I have switched line 13 to use $workdir instead